sdk/nodejs/api/client.gen.ts

```diff
@@ -382,9 +382,13 @@ export class Container extends BaseClient {
 
   /**
    * This container after executing the specified command inside it
-   *
-   * @param opts optional params for exec
-   *
+   * @param opts.args Command to run instead of the container's default command
```

Comment:
This is awesome to have explicit opts args 👍 !!
When I look into `client.gen.ts`, those explicit params are only for `exec`, `withExec`. 
Is this something that we can generalize to every method that takes opts ? For instance:
```ts
build(
    context: DirectoryID | Directory,
    opts?: ContainerBuildOpts
  )
  ```
This can be done in a follow up PR, this is not a blocker as this PR is getting old and should be merged.

Reply:
There is a good reason for this: There is no documentation on every other params for functions in the GraphQL documentation. Hence: https://github.com/dagger/dagger/issues/4129 and https://github.com/dagger/dagger/pull/4240
Good thing, adding doc in the schema will add doc in our code.

Reply:
Make sense. Thanks for the clarification